R/expr-binary.R

```diff
@@ -57,3 +149,46 @@ expr_bin_encode <- function(encoding) {
     )
   })
 }
+
+#' Get the size of binary values in the given unit
+#'
+#' @param unit Scale the returned size to the given unit. Can be `"b"`, `"kb"`,
+#' `"mb"`, `"gb"`, `"tb"`, or their full names (`"kilobytes"`, etc.).
+#'
+#' @inherit as_polars_expr return
+#'
+#' @examples
+#' df <- pl$DataFrame(
+#'   name = c("black", "yellow", "blue"),
+#'   code_hex = as_polars_series(c("000000", "ffff00", "0000ff"))$cast(pl$Binary)
+#' )
+#'
+#' df$with_columns(
+#'   n_bytes = pl$col("code_hex")$bin$size(),
+#'   n_kilobytes = pl$col("code_hex")$bin$size("kb")
+#' )
+expr_bin_size <- function(unit = "b") {
+  wrap({
+    arg_match0(unit, values = c(
+      "b", "bytes",
+      "kb", "kilobytes",
+      "mb", "megabytes",
+      "gb", "gigabytes",
+      "tb", "terabytes"
+    ))
```

Comment:
It seems that only `('b', 'kb', 'mb', 'gb', 'tb')` are sufficient to support.
This is a small number and can be added to the default argument, right?
https://github.com/pola-rs/polars/blob/d8dde13cac8a2d570b2f41e16c2670b09836ee5f/py-polars/polars/expr/binary.py#L262-L263

Reply:
Indeed, they probably changed it recently. I'll fix that and add the choices to the default arg